switch_mod/utilities.py

```diff
@@ -223,11 +221,9 @@ def post_solve(model, outputs_dir=None):
     # (the latter occurs when there are problems with licenses, etc)
     
     # replace the old _save_results function
```

Comment:
Hopefully any hypothetical user would also search the git logs for save_results, find this commit, and figure out what happened. I don't know, maybe Mark has another take on that?

The TODO comment looks fine to me. In my opinion, those work great for short notes, while github issues are better for longer explanations or discussions.